src/components/editor/editor.js

```diff
@@ -47,21 +47,38 @@ class PageContainer extends React.Component {
   //A ref that points to the editor area to make it droppable. Could (and probably should) just select this id.
   dropRef = React.createRef()
 
-  //Two key presses are listed for: 
+  //Two key presses are listed for:
   //Enter - when enter is pressed, the whole doc is scanned for our regexes and rendered.
-  //Space - some styles we want to cancel after a space. That happens here too. 
+  //Space - some styles we want to cancel after a space. That happens here too.
   handleKeyDown = (e) => {
     if (e.key === 'Enter') {
       this.renderEverything()
     }
     if (e.keyCode === 32) {
       this.clearStyles()
     }
+    // Styling
+    if (e.metaKey === true && e.key === 'b') {
+      e.preventDefault();
+      this.surroundText('**'); // Bold
+    }
+    if (e.metaKey === true && e.key === 'i') {
+      e.preventDefault();
+      this.surroundText('*'); // italic
+    }
+    if (e.metaKey === true && e.shiftKey === true && e.key === 'x') {
+      e.preventDefault();
+      this.surroundText('~~'); // strikethrough
+    }
+    if (e.metaKey === true && e.ctrlKey === true && e.key === 'c') {
+      e.preventDefault();
+      this.surroundText('`'); // code
+    }
```

Comment:
For us windows folks, also include the e.ctrlKey. The meta key on a windows keyboard is the windows key and it really doesn't work.

Reply:
Ahhh yeah, good idea

Reply:
Change all meta to ctrl and revised shortcut for consistency